scripts/create-substrate-fork.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+BASE_BRANCH=${2:-"moonbeam-polkadot-v0.9.12"}
+NEW_BRANCH=$1
+
```

Comment:
Adding a check those values are not empty and otherwise display a "usage" would be helpful

Reply:
done